crates/bevy_ecs/src/world/mod.rs

```diff
@@ -1634,12 +1633,11 @@ impl World {
         let change_tick = self.change_tick();
         let (ptr, ticks) = self.get_non_send_with_ticks(component_id)?;
 
-        let ticks = unsafe {
+        let ticks =
             // SAFETY: This function has exclusive access to the world so nothing aliases `ticks`.
             // - index is in-bounds because the column is initialized and non-empty
             // - no other reference to the ticks of the same row can exist at the same time
-            TicksMut::from_tick_cells(ticks, self.last_change_tick(), change_tick)
-        };
+            unsafe { TicksMut::from_tick_cells(ticks, self.last_change_tick(), change_tick) };
```

Comment:
IMO putting the SAFETY comment between the let and unsafe block is ugly but rustfmt won't have it any other way.

The only other solve here would be to see if lifting the `self.last_change_tick()` into a local variable got the line length below the limit which didn't seem worth it.